CONTRIBUTING.md

```diff
@@ -66,7 +66,7 @@ Every contribution is much appreciated!
 
 ## Updating documentation
 
-The documentation resides under the `./docs` directory.
+The documentation resides under the [./docs](./docs) directory.
```

Comment:
TODO:

these relative links i.e. './' only work because CONTRIB... is at repo root.

if these were every copy-paste to another file they would suddenly break. Best to use absolute links I'd say.

Reply:
I'm unsure what you mean by absolute links, but would they work in Obsidian?

What if the links looked like this:

```text
- [./docs](docs/)
- [../docs](docs/)
```

I'm not too worried if the links need updating of text is copied or moved around, as relatively few people will be updating this text, and if a link happens to be broken it will be easy to fix.

Also, the pool of people reading them is rather smaller than the Tasks plugin user base - so the bar for 'good enough' is a lot lower than that for the main plugin docs...